docs/SSO/oauth2_flow_diagram.md

```diff
@@ -0,0 +1 @@
+![Oauth2_flow_diagram.png](../../assets/images/Oauth2_flow_diagram.png)
```

Comment:
Why is the image being displayed in a separate file? You should probably move it to be included inside the guide.